### PR TITLE
Report commit statuses by using both github statuses and check-suite …

### PR DESCRIPTION
**Pull request by cpennington**
…results

### CODE REVIEW

File: tubular/github_api.py [hunk 1]
```diff
@@ -307,6 +308,97 @@ def get_commit_combined_statuses(self, commit):
...
+        Returns:
+            Json representing the check suites
+        """
+        if isinstance(commit, six.string_types):
+            commit = self.github_repo.get_commit(commit)
+        elif isinstance(commit, GitCommit):
+            commit = self.github_repo.get_commit(commit.sha)
+        elif not isinstance(commit, Commit):
+            raise UnknownObjectException(
+                500, "commit is neither a valid sha nor github.Commit.Commit object."
```
Comment (crice100) [inline tubular/github_api.py:330]: Missing an "or" (GitCommit or Commit)?

File: tubular/github_api.py [hunk 2]
```diff
@@ -307,6 +308,97 @@ def get_commit_combined_statuses(self, commit):
...
+            suite['app']['name']: (
+                suite.get('conclusion', 'pending').lower(),
+                suite['url']
+            )
+            for suite in check_suites['check_suites']
+        })
+
+        return results
+
+    def filter_validation_results(self, results, exclude_apps='', include_apps=''):
```
Comment (crice100) [inline tubular/github_api.py]: Where is this called? Should we add a test for the include/exclude?

File: tubular/github_api.py (hunk 1 above)
Comment (cpennington) [reply tubular/github_api.py:330]: I'm not sure what you mean. Where you would put an `or` in that sentence?

Comment (crice100) [reply tubular/github_api.py:330]: Minor (possibly irrelevant) point: I read the doc to say "commit could be 1 of these 3 things: _string_, _GitCommit_ or _Commit_". But the exception message talks about "_github.Commit.Commit_", which looks to be a typo.

File: tubular/github_api.py (hunk 2 above)
Comment (cpennington) [reply tubular/github_api.py]: I'll delete it. It's not currently used.

**Review by crice100 — APPROVED** [review]

**Pull request merged**